setup.py

```diff
@@ -4,13 +4,13 @@
     from distutils.core import setup
 
 
-setup(name='grading',
+setup(name='abc-classroom',
       version='0.0.1',
       description='Authoring and grading of notebook assignments',
       long_description='Authoring and grading of notebook assignments',
       license='BSD',
       author='UC Boulder Earthlab',
-      packages=['grading'],
+      packages=['abcclassroom'],
```

Comment:
hey @betatim i fixed the spelling here of classroom. Was that intentional? i don't see my commit int he PR like you had provided me ... so hopefully i did this right. please have a look. i will then merge this if you are ok with the small fix. if not i'll remove the o's :)